src/controllers/controllerpresetfilehandler.cpp

```diff
@@ -87,6 +87,8 @@ void ControllerPresetFileHandler::parsePresetInfo(
         return;
     }
 
+    int iControllerEngineVersion = root.attribute("controllerEngineVersion", 0).toInt();
```

Comment:
DEBUG_ASSERT(iControllerEngineVersion >= 0)
Not a replacement for proper input validation but better than nothing

Reply:
I don't think we need this anymore since 2.3 took so long that 2.4 will be released with the latest Ubuntu LTS having Qt 5.12.